apps/web/src/pages/templates/components/TemplateEditorFormProvider.tsx

```diff
@@ -42,7 +42,7 @@ const makeStep = (channelType: StepTypeEnum, id: string): IFormStep => {
       subject: '',
       type: channelType,
       content: channelType === StepTypeEnum.EMAIL ? defaultEmailBlocks : '',
-      contentType: 'editor',
+      contentType: 'customHtml',
```

Comment:
@gitstart this change will modify all existing channel which is something we don't want. This fix should only apply to the "Chat" channel and not on all available channels 🙏

Reply:
@scopsy sorry for the delay on this. The latest updates resolves this. Here is how it renders on slack:

![image](https://github.com/novuhq/novu/assets/1501599/60ec6bed-35ec-4156-bbc5-7a7f5dff61af)